src/rgw/rgw_kafka.cc

```diff
@@ -42,7 +42,7 @@ inline int rd_kafka_err2errno(rd_kafka_resp_err_t err) {
   case RD_KAFKA_RESP_ERR_MSG_SIZE_TOO_LARGE:
     return EMSGSIZE;
   case RD_KAFKA_RESP_ERR__QUEUE_FULL:
-    return ENOBUFS;                                                                                                                                                                                                                           
+    return ENOBUFS;
```

Comment:
what is this change?

Reply:
I am not sure but it seems like some whitespaces have been removed - probably by my IDE.